env.daint.sh

```diff
@@ -92,12 +92,12 @@ setCppEnvironment()
     fi
     
     # standard modules (part 1)
-    module load cmake
-    module unload gcc
-    module load gcc/4.8.2
+    module load cmake    
+    module swap gcc/4.8.2 gcc/4.9.3
     if [ "${target}" == "gpu" ] ; then
-        module load craype-accel-nvidia35
-        module swap cudatoolkit/7.0.28-1.0502.10742.5.1
+        module load cudatoolkit/7.0.28-1.0502.10742.5.1
+        module load cray-libsci_acc/3.3.0
```

Comment:
The problem is that by default a ton of cray modules are already loaded. With these we just update the modules.